proxy/http/HttpTransact.h

```diff
@@ -905,8 +905,9 @@ class HttpTransact
     int next_hop_scheme; // out
     int orig_scheme;     // pre-mapped scheme
     int method;
-    int cause_of_death_errno; // in
-    HostDBInfo host_db_info;  // in
+    int cause_of_death_errno;     // in
+    Ptr<HostDBInfo> hostdb_entry; // Pointer to the entry we are referencing in hostdb-- to keep our ref
```

Comment:
right now-- nowhere. This is simply to have the transaction maintain a refcount to the HostDBInfo. Its a bit complicated as there is another member `host_db_info` which is the exact record (child or whatnot) that the transaction is using.